quickstarts/notification-channels/webhook-notifications/config.yml

```diff
@@ -0,0 +1,32 @@
+name: webhook-notifications
+
+title: Webhook Notifications
+
+description:  |
+  
+  Webhook is a method of augmenting or altering the behavior of a web page or web application with custom callbacks.
+
+  Check out our documentation to set up a Webhook notification channel and provide fast and consistent ways for the right personnel to be notified about incidents
+
+summary: |
+  Send your New Relic alerts via Webhook
+logo: logo.png
+level: New Relic
+
+authors:
+  - Matan Moser (New Relic)
+  - Ismail Azam (New Relic)
+
+keywords:
+  - webhook
+  - notifications
+  - AIOps
+  - Incident Intelligence
+  - Destinations
+
+documentation:
```

Comment:
Won't this add a tile to the data source tab? Which I thought we agreed would confuse the user and the idea was to just link to these doc URLS from within the markdown description.